src/main/config/parameter_group_ids.h

```diff
@@ -0,0 +1,79 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+// FC configuration
+#define PG_FAILSAFE_CONFIG 1
+#define PG_BOARD_ALIGNMENT 2
+#define PG_GIMBAL_CONFIG 3
+#define PG_MOTOR_MIXER 4
+#define PG_BLACKBOX_CONFIG 5
+#define PG_MOTOR_AND_SERVO_CONFIG 6
+#define PG_SENSOR_SELECTION_CONFIG 7
+#define PG_SENSOR_ALIGNMENT_CONFIG 8
+#define PG_SENSOR_TRIMS 9
+#define PG_GYRO_CONFIG 10
+#define PG_BATTERY_CONFIG 11
+#define PG_CONTROL_RATE_PROFILES 12
+#define PG_SERIAL_CONFIG 13
+#define PG_PID_PROFILE 14
+#define PG_GTUNE_CONFIG 15
+#define PG_ARMING_CONFIG 16
+#define PG_TRANSPONDER_CONFIG 17
+#define PG_SYSTEM_CONFIG 18
+#define PG_FEATURE_CONFIG 19
+#define PG_MIXER_CONFIG 20
+#define PG_SERVO_MIXER 21
+#define PG_IMU_CONFIG 22
+#define PG_PROFILE_SELECTION 23
+#define PG_RX_CONFIG 24
+#define PG_RC_CONTROLS_CONFIG 25
+#define PG_MOTOR_3D_CONFIG 26
+#define PG_LED_STRIP_CONFIG 27
+#define PG_COLOR_CONFIG 28
+#define PG_AIRPLANE_ALT_HOLD_CONFIG 29
+#define PG_GPS_CONFIG 30
+#define PG_TELEMETRY_CONFIG 31
+#define PG_FRSKY_TELEMETRY_CONFIG 32
+#define PG_HOTT_TELEMETRY_CONFIG 33
+#define PG_NAVIGATION_CONFIG 34
```

Comment:
It is. I copied over the `#defines` from Cleanflight so that we do not re-use Cleanflight numbers. PGs are 12-bit numbers so there are plenty of them. New ones for iNav should have the form `PG_INAV_xxxx` and should have different numbers.

We need to coordinate this with @hydra for Cleanflight and @borisbstyle for betaflight.

Probably the easiest thing to do is to allocate a range of numbers for iNav and for betaflight.